public/api/conf/1.0/application.raml

```diff
@@ -68,26 +68,36 @@ documentation:
                   description: Valid XML Payload
                   example: |
                     <?xml version="1.0" encoding="UTF-8"?>
-                    <soapenv:Envelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-                      xmlns:soapenc="http://schemas.xmlsoap.org/soap/encoding/"
-                      xmlns:xsd="http://www.w3.org/2001/XMLSchema"
-                      xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/">
-                       <soapenv:Header/>
-                       <soapenv:Body>
-                          <n1:InventoryLinkingImportsDeclarationStatusNotification xmlns:n1="http://gov.uk/customs/inventoryLinkingImport/v1">
-                             <n1:declarationMethod>2</n1:declarationMethod>
-                             <n1:dateOfEntry>2018-11-15T11:54:27.000Z</n1:dateOfEntry>
-                             <n1:entryNumber>18GBCN36B8HDVFGVR6</n1:entryNumber>
-                             <n1:entryVersionNumber>2</n1:entryVersionNumber>
-                             <n1:goodsArrivalDeclaration>Y</n1:goodsArrivalDeclaration>
-                             <n1:ics>14</n1:ics>
-                             <n1:inventoryConsignmentReference>GB/1234-99999886JCP</n1:inventoryConsignmentReference>
-                             <n1:roe>0</n1:roe>
-                             <n1:traderReference>8GB025115100018-123456ABABAB1</n1:traderReference>
-                             <n1:totalPackages>55</n1:totalPackages>
-                          </n1:InventoryLinkingImportsDeclarationStatusNotification>
-                       </soapenv:Body>
-                    </soapenv:Envelope>
+                    <_2:MetaData xmlns:_2="urn:wco:datamodel:WCO:DocumentMetaData-DMS:2"><_2:WCODataModelVersionCode>3.6</_2:WCODataModelVersionCode><_2:WCOTypeName>RES</_2:WCOTypeName><_2:ResponsibleCountryCode/><_2:ResponsibleAgencyName/><_2:AgencyAssignedCustomizationCode/><_2:AgencyAssignedCustomizationVersionCode/><_2_1:Response xmlns:_2_1="urn:wco:datamodel:WCO:RES-DMS:2">
```

Comment:
_2 as namespace. This is nuts.

Reply:
i said the same to Nick. Prefer `_2_1` myself!